packages/viewer/src/components/viewer/viewer.tsx

```diff
@@ -231,6 +231,11 @@ export class Viewer {
    */
   @Prop() public noDefaultLights = false;
 
+  /**
+   * Specifies experimental rendering options. For Vertex use only.
+   */
```

Comment:
Can we add this doc metadata so this field is not documented.

```ts
/**
 * @private
 * @internal
 */
```

Reply:
If I specify `@internal`, the reference to the property in these lines in connect-app don't work. Is there another way to do this to avoid that problem? https://github.com/Vertexvis/connect-app/pull/483/files#diff-d70ac4a03197f1c59a02349a3466d3dfc5d17f805caf796d5a7a03ab74cd7894R143